fuse.py

```diff
@@ -483,6 +491,11 @@ def _wrapper(func, *args, **kwargs):
             print_exc()
             return -EFAULT
 
+    def _handle_nopath(self, path):
```

Comment:
Yes, I'd prefer "decode" in the method name to make it clear it is decoding the path